docs/configuration-ini.md

```diff
@@ -255,6 +255,48 @@ bias_initializer = zeros
 Activation = softmax
 ```
 
+### Backbone section
+
+Describe another model, termed as backbone, to be used used in the model described by the current ini file.
```

Comment:
Maybe `Backbone section describes ... `?? and `used` is used twice. `to be used used`,

Reply:
done.